src/main/java/lotto/model/LottoNumber.java

```diff
@@ -0,0 +1,43 @@
+package lotto.model;
+
+import static lotto.LottoNumberConstants.LOTTO_NUMBER_MAX;
+import static lotto.LottoNumberConstants.LOTTO_NUMBER_MIN;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private final int number;
+
+    public LottoNumber(int number) {
+        validateNumberInRange(number);
+        this.number = number;
+    }
+
+    private void validateNumberInRange(int number) {
+        if (number < LOTTO_NUMBER_MIN.value() || number > LOTTO_NUMBER_MAX.value()) {
+            throw new IllegalArgumentException("로또 번호는 1부터 45 사이의 수여야 합니다.");
```

Comment:
아래와 같이 하면 로또의 숫자 범위가 변경됐을 때 (그럴일은 없겠지만요... 미래는 모르는것이니)
수정할 부분이 줄어들겠네요!

```suggestion
            throw new IllegalArgumentException(String.format("로또 번호는 %d부터 %d 사이의 수여야 합니다.", LOTTO_NUMBER_MIN, LOTTO_NUMBER_MAX));
```

Reply:
유지보수 고려를 위해서는 메세지 formatting 고려하는 것 좋은 것 같아요!
놓쳤었는데 감사합니다!